src/stream_reader_group.rs

```diff
@@ -99,6 +105,36 @@ impl PyObjectProtocol for StreamReaderGroupConfig {
     }
 }
 
+#[cfg(feature = "python_binding")]
+#[pyclass]
+#[derive(Clone)]
+pub(crate) struct ReaderStreamCut {
```

Comment:
This is a confusing name. StreamCuts are for the whole stream and not specific to a reader. We can call this StreamCut and alias the StreamCut that is being imported to something else

Reply:
Done.